azurefunctions-extension-balo/src/azure.functions/Module.md

```diff
@@ -0,0 +1,209 @@
+# Module Overview
+
+Annotation based Azure Functions extension implementation for Ballerina. 
+
+## Azure Setup
+
+* An Azure "Function App" needs to be created in a given resource group with the following requirements
+   - Runtime stack - "Java"
```

Comment:
We have to mention the version also here now, since in the Azure Function App UI, once you select "Java", thereafter you can select either Java 8 or 11, earlier it was only 8 available. So for the moment, let's mention Java 8 here, but later on when we move Swan Lake to Java 11, this should be updated. The 1.2.x branch should mention Java 8.

Reply:
Also, please test the extension with Swan Lake. I saw that we had language changes that affected the module.

Reply:
I only copied what was in the README.md to the Module.md. Didn't really go through it. 

Will you be able to send a separate PR with the content updated if its okay ?

Reply:
The build for this PR passed successfully with the swan lake 2.0.0-Preview3-SNAPSHOT version with tests - https://github.com/ballerina-platform/module-ballerinax-azure.functions/pull/15/checks?check_run_id=898167956

Do we need to run additional tests ?

Reply:
Sure, I can update the content later. And yes, if the tests pass properly, it should be okay. I will also test with the actual environment later.